drivers/hyperv/powershell.go

```diff
@@ -69,6 +69,36 @@ func hypervAvailable() error {
 }
 
 func isAdministrator() (bool, error) {
+	var hypervAdmin bool
```

Comment:
possibly because the authors (well this one at least) are new to go and this was a drive by submission :smile:   Made the changes - but not tested, assuming they are functionally equivalent.